plugins/src/webrtcsink/imp.rs

```diff
@@ -1749,29 +1744,28 @@ impl WebRTCSink {
         let mut state = element.imp().state.lock().unwrap();
 
         if let Some(consumer) = state.consumers.get_mut(peer_id) {
+            // Start adding some FEC when the bitrate > 2Mbps as we found experimentally
```

Comment:
Maybe add that comment around the constant definition?

Reply:
Makes sense. Will do :)